src/components/application_manager/include/application_manager/policies/policy_handler.h

```diff
@@ -583,6 +597,19 @@ class PolicyHandler : public PolicyHandlerInterface,
    */
   void OnAppsSearchCompleted(const bool trigger_ptu) OVERRIDE;
 
+  /**
+   * @brief New application was added to application list
+   * @param new_app_id app_id for this application
+   * @param policy_id policy_id for this application
+   */
+  void OnAddedNewApplicationToAppList(const uint32_t new_app_id,
```

Comment:
@ydementieiev Move description to iface

Reply:
@AByzhynar  leaved description only in iface [c04c115ce](https://github.com/smartdevicelink/sdl_core/commit/c04c115ced5d0ea1dfcc5bb53fc9c01d6d09a998)